a_socials/urls.py

```diff
@@ -20,7 +20,6 @@
 from api import views
 
 urlpatterns = [
-    url(r'^$', views.HomeView.as_view(), name='homepage'),
```

Comment:
I think this view is meant to serve the homepage. Removing it will make the page inaccessible for `REACT`.

Reply:
Oh seen! I see what you were trying to do!